rocky/rocky/settings.py

```diff
@@ -397,15 +398,18 @@ def immutable_file_test(path, url):
     MIDDLEWARE += ["csp.middleware.CSPMiddleware"]
     INSTALLED_APPS += ["csp"]
 
-CSP_DEFAULT_SRC = ["'none'"]
-CSP_IMG_SRC = ["'self'"]
-CSP_FONT_SRC = ["'self'"]
-CSP_STYLE_SRC = ["'self'"]
-CSP_FRAME_ANCESTORS = ["'none'"]
-CSP_BASE_URI = ["'none'"]
-CSP_FORM_ACTION = ["'self'"]
-CSP_INCLUDE_NONCE_IN = ["script-src"]
-CSP_CONNECT_SRC = ["'self'"]
+    CONTENT_SECURITY_POLICY = {
```

Comment:
CSP_INCLUDE_NONCE_IN seems to have fallen of the new directives list. possibly because it not standard django as much as the rest?

Reply:
`CSP_INCLUDE_NONCE_IN` used to be a configuration in 3.8 to enable dynamically generated nonce in the configured directives ("script-src" in this case). With 4.0 this seems default behavior, but I had to also include the `script-src` directive explicitly